bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="chatgpt" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>ChatGPT Binding</name>
+	<description>This is the binding for ChatGPT integration.</description>
```

Comment:
Can we add `connection` here?
```suggestion
	<description>This is the binding for ChatGPT integration.</description>
	<connection>cloud</connection>
```

See https://next.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions